recipes/pylabeladjust/meta.yaml

```diff
@@ -37,7 +37,7 @@ test:
 about:
   home: https://github.com/MNoichl/pylabeladjust
   summary: "`pylabeladjust` is a port of Gephi's Label-Adjust algorithm to python."
-  license: GPL-3.0
+  license: GPL-3.0-only
```

Comment:
It looks like an "or-later" version. See https://github.com/MNoichl/pylabeladjust/blob/bcece89ca79eddb8e4cdde9cc21a4ccc4bca4453/LICENSE#L639